src/tests/ftest/dfuse/caching_check.yaml

```diff
@@ -34,7 +34,7 @@ ior:
   transfer_size: '1M'
   block_size: '64M'
   dfs_oclass: "EC_2P1G1"
-  read_x: 5  # 500%
+  read_x: 4.5  # 450%
```

Comment:
This is fine, but FYI we could lower more if needed

Reply:
It passed 10/10 with this so I'm OK with the new value.